src/schema/objects/metadata.yaml

```diff
@@ -3392,6 +3392,23 @@ SubjectArtefactDescription:
     If this field is set to `"n/a"`, it will be interpreted as absence of major
     source of artifacts except cardiac and blinks.
   type: string
+TablePosition:
+  name: TablePosition
+  display_name: Table Position
+  description: |
+    The table position, relative to an implementation-specific reference point,
+    often the isocenter. Values must be an array of three distances in
+    millimeters. The coordinates are respectively right, anterior and superior
+    (RAS+) as if it were acquired in head first supine (HFS) orientation. In
```

Comment:
should we maybe say something about the first two values, which, in most cases I guess will be zero